rust/connlib/clients/apple/src/lib.rs

```diff
@@ -136,8 +137,8 @@ impl Callbacks for CallbackHandler {
     }
 }
 
-fn init_logging(log_dir: PathBuf) -> WorkerGuard {
-    let (file_layer, guard) = file_logger::layer(log_dir.clone());
+fn init_logging(log_dir: PathBuf, log_string: String) -> WorkerGuard {
```

Comment:
I don't think `log_string` is a very good name for this variable. How about `log_filter`?